packages/client/src/Dashboard.tsx

```diff
@@ -271,12 +281,27 @@ function SelectedActionsBar({
       >
         <Toolbar className={classes.toolbar}>
           <span className={classes.title} />
-          <IconButton aria-label="archive" onClick={onArchive}>
-            <ArchiveIcon />
-          </IconButton>
-          <IconButton aria-label="star" onClick={onFlag}>
-            {isStarred ? <StarBorder /> : <StarIcon />}
-          </IconButton>
+          <Tooltip
+            title="Archive Selected Conversation"
```

Comment:
These tooltips are great! But please pluralize "Conversations" for the list view actions.